src/com/company/DataGenerator.java

```diff
@@ -29,9 +29,13 @@ public static void createCustomer(){
 
         Customer customer1 = new Customer(UUID.randomUUID(),"ozzy","ozzy@cydeo.com",customer1AddressList);
         Customer customer2 = new Customer(UUID.randomUUID(),"mike","mike@gmail.com");
+        Customer customer3=new Customer(UUID.randomUUID(),"Kicchi","kicchi@gmail.com");
+
```

Comment:
Can you please remove empty line?

Reply:
I removed empty line. Could you review my pull request please?